Makefile

```diff
@@ -2,7 +2,6 @@
 update-version:
 	@echo "$(VERSION)" > VERSION
 	@perl -pi -e 's|"version": "[.\-\d\w]+"|"version": "$(VERSION)"|' package.json
-	@find . -name 'package.json' -exec perl -pi -e 's|"stripe": "[\^\d\w.]*?"|"stripe": "^$(VERSION)"|' {} +
```

Comment:
I realized this is a bad idea, the package won't be available between the time we update the version and release the actual package to NPM. If we update samples during that time we'll fail the build.